tenets/codelingo/go/only-use-must-in-init/codelingo.yaml

```diff
@@ -0,0 +1,27 @@
+funcs:
```

Comment:
I feel like this tenet may benefit from a small doc section explaining why we enforcing this. Mention unexpected panics at runtime.